open-source-software-submissions/editor-in-chief-guide.md

```diff
@@ -1,19 +1,131 @@
-- Monitors the review process pace and reminds other editors to move packages along as needed.
+If the EiC needs to step down at any time before the agreed-upon 
+duration of their tenure, pyOpenSci requests and appreciates 2 weeks notice
+to support finding a new EiC.
 
-## Editor in Chief Checklist
-- Tag issue with `1/editor-checks` tag and assign a main editor.
+### The Editor in Chief fulfills the following roles:
+
+- Watches all issues posted to the software-review repo (either subscribe to repo notifications on GitHub, or watch the `#software-review-feed` channel on Slack).
+- Performs an initial set of editor checks on new package submissions (see template for these checks below)
+- Raises scope/overlap issue with all editors if they see an ambiguous submission. 
+This may also be done by handling editors if appropriate (see note below). If 
+the scope of a package is in question, the EiC should post to the pyOpenSci 
+Slack `software-review` channel, tagging all editors.
+- Assigns package submissions to other editors (including themselves if they have bandwidth), to handle. Assigning may be based upon domain background or just who has bandwidth to lead the review. 
+- Responds to pre-submission inquiries posted to the software-review repository 
+and similarly pings editors in the `software-review slack channel` if discussion 
+is needed. Any editor should feel free to communicate with package authors as it 
+makes sense. See [this section](#responding-to-out-of-scope-submissions) about 
+how to respond to out-of-scope (pre-) submissions.
+- Responds to review referrals from JOSS or other publication partners.
+- Monitors pace of review process and reminds other editors to move packages along as needed.
+- Requests a new EiC when their rotation is up (set a calendar reminder ahead of your expected end date and ask for volunteers in the editors’ Slack channel)
+
+This Editor in Chief position rotates between different pyOpenSci editors. 
+
+```{note}
+In some cases, if the EiC is unable to support a particular package due to 
+conflicts OR if they simple believe another editor is better suited to assess 
+the scope and readiness of a package to be reviewed, they may opt to assign an editor to perform initial checks.
+```
+
+## Editor in Chief checklist
+
+When a new package is submitted for review, the EiC will:
+
+- Tag the issue with `1/editor-checks` tag 
+- Next, they will use the template below in the issue to check whether the package has 
+the bare minimum requirements to initiate a review (or they will assign that task to an editor as stated above). 
+- SURVEY data: To ensure we have information from the authors, please be sure that they filled out our survey. Thank them graciously for doing this. They can skip sections of it if they wish, but we do need their contact information and we do want to track their experience with out review process and organization  
+
+
```

Comment:
```suggestion
(editor-checklist-template)=
```

Reply:
so @NickleDave this is where i'm confused... in theory those headers should automagically be creating targets on the page. but you are creating a target here. this is where i'm confused with the jupyter book build and how links work. i have an discussion going on with Chris over in the executable books org about this.  Also can i safely call that target from another page?